source/StateManager.lua

```diff
@@ -160,6 +168,22 @@ function StateManager:new(list, logger)
     return o
 end
 
+function StateManager:travelToTime(timestamp)
+
+    if self.timeTraveling == nil or self.timeTraveling > timestamp then
+        self.timeTraveling = timestamp
+        self:restart()
+    else
+        self.timeTraveling = timestamp
+    end
+end
+function StateManager:stopTimeTravel()
+    self.timeTraveling = nil
```

Comment:
Shouldn't you add either `restart` after this or continuing calculation? after `timeTraveling` goes to `nil` the mutators are either not fired or the `onUpdate` events are not called with `lag` and `uncommited` at `0,0`

Reply:
In my testing they are / were.
Timetravel is a limit used by the state updater, which runs on an interval. So stopping time travel is just removing the limit, nothing else changes. 
Thus the next interval we will again do mutations until we reach the end...

Regardless of whether that part works though, there is no practical reason why a restart is required, since we're moving forward in time only.

Reply:
Actually could it be that the `>`  is supposed to be a `<`?
Since when we're already time travelling, if we travel further back in time a reset is needed..